#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_independent_working_testfiles.dpatch by Daniel Leidert <daniel.leidert@wgdd.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: We install test-files into the examples dir of our documentation
## DP: package. So we need to use the right path.

@DPATCH@
diff -urNad gnome-chemistry-utils-0.4.1/tests/testbonobocontrol.c /tmp/dpep.rzIqRX/gnome-chemistry-utils-0.4.1/tests/testbonobocontrol.c
--- gnome-chemistry-utils-0.4.1/tests/testbonobocontrol.c	2005-03-06 21:43:43.000000000 +0100
+++ /tmp/dpep.rzIqRX/gnome-chemistry-utils-0.4.1/tests/testbonobocontrol.c	2005-03-21 20:36:26.830122704 +0100
@@ -53,7 +53,7 @@
 	/*
 	 *Loading data
 	 */
-	uri = "file://"SRCDIR"/tests";
+	uri = "file://"DATADIR"/doc/libgchemutils-doc/examples";
 	bu = bonobo_widget_get_objref (BONOBO_WIDGET(control));
 	bs = Bonobo_Unknown_queryInterface(bu,"IDL:Bonobo/PersistStream:1.0",&ev);
 	storage = bonobo_get_object (uri, "IDL:Bonobo/Storage:1.0", &ev);
diff -urNad gnome-chemistry-utils-0.4.1/tests/testgtkchem3dviewer.c /tmp/dpep.rzIqRX/gnome-chemistry-utils-0.4.1/tests/testgtkchem3dviewer.c
--- gnome-chemistry-utils-0.4.1/tests/testgtkchem3dviewer.c	2004-04-07 16:56:22.000000000 +0200
+++ /tmp/dpep.rzIqRX/gnome-chemistry-utils-0.4.1/tests/testgtkchem3dviewer.c	2005-03-21 20:36:31.230453752 +0100
@@ -26,7 +26,7 @@
 		 NULL);
 
 	if (argc >= 2) uri = argv[1];
-		else uri = "file://"SRCDIR"/tests/methane.xyz";
+		else uri = "file://"DATADIR"/doc/libgchemutils-doc/examples/methane.xyz";
 	
 	viewer = gtk_chem3d_viewer_new(uri);
 	gtk_container_add(GTK_CONTAINER(window), viewer);
diff -urNad gnome-chemistry-utils-0.4.1/tests/testgtkcrystalviewer.c /tmp/dpep.rzIqRX/gnome-chemistry-utils-0.4.1/tests/testgtkcrystalviewer.c
--- gnome-chemistry-utils-0.4.1/tests/testgtkcrystalviewer.c	2005-03-06 21:43:43.000000000 +0100
+++ /tmp/dpep.rzIqRX/gnome-chemistry-utils-0.4.1/tests/testgtkcrystalviewer.c	2005-03-21 20:37:30.154495936 +0100
@@ -12,6 +12,7 @@
 {
 	GtkWidget *window;
 	GtkWidget *viewer;
+	gchar* uri;
 	const char* filename;
 	xmlDocPtr xml;
 	gtk_init (&argc, &argv);
@@ -22,9 +23,9 @@
 		 GTK_SIGNAL_FUNC(gtk_main_quit),
 		 NULL);
 
-	if (argc >= 2) filename = argv[1];
-		else filename = SRCDIR"/tests/nickel.gcrystal";
-	xml = xmlParseFile(filename);
+	if (argc >= 2) uri = argv[1];
+		else uri = "file://"DATADIR"/doc/libgchemutils-doc/examples/nickel.gcrystal";
+	xml = xmlParseFile(uri);
 	
 	viewer = gtk_crystal_viewer_new(xml->children);
 	gtk_container_add(GTK_CONTAINER(window), viewer);
